benchmark/cublas/blas3/trsm_batched.cpp

```diff
@@ -0,0 +1,292 @@
+/**************************************************************************
+ *
+ *  @license
+ *  Copyright (C) Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename trsm_batched.cpp
+ *
+ **************************************************************************/
+
+#include "../utils.hpp"
+
+// Convert batch_type=strided to interleaved on the host
+template <typename scalar_t>
+std::vector<scalar_t> strided_to_interleaved(const std::vector<scalar_t>& input,
```

Comment:
I don't think `cublas` has support for `interleaved` data layout for `trsm`. Can we modify this benchmark to support only `strided` layout?